rotkehlchen/tests/unit/decoders/test_liquity.py

```diff
@@ -987,3 +987,209 @@ def test_stability_pool_withdrawal(database, ethereum_inquirer, ethereum_account
+            timestamp=TimestampMS(1664055431000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.STAKING,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_LUSD,
+            balance=Balance(amount=FVal('87.49718915849563905')),
+            location_label=user_address,
+            notes='Deposit 87.49718915849563905 LUSD in liquity\'s stability pool',
+            counterparty=CPT_LIQUITY,
+            address=string_to_evm_address('0x7815beb98a927565eA43b5854644392F21dA0021'),
+        ), EvmEvent(
+            event_identifier=evmhash,
+            sequence_index=292,
+            timestamp=TimestampMS(1664055431000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.APPROVE,
+            asset=A_LUSD,
+            balance=Balance(amount=FVal('1.157920892373161954235709850E+59')),
+            location_label=user_address,
+            notes='Set LUSD spending approval of 0x0c3ce74FCB2B93F9244544919572818Dc2AC0641 by 0x7815beb98a927565eA43b5854644392F21dA0021 to 115792089237316195423570985000000000000000000000000000000000',  # noqa: E501
+            address=string_to_evm_address('0x7815beb98a927565eA43b5854644392F21dA0021'),
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x0c3ce74FCB2B93F9244544919572818Dc2AC0641']])
+def test_ds_proxy_liquity_withdraw(database, ethereum_inquirer, ethereum_accounts):
+    user_address = ethereum_accounts[0]
+    evmhash = deserialize_evm_tx_hash('0xdac8d9273a17b00fb81e89839d0c974e393db406a641552051419646b902c4b3')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    expected_events = [
+        EvmEvent(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=TimestampMS(1665138455000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.002590429704686116')),
+            location_label=user_address,
+            notes='Burned 0.002590429704686116 ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            event_identifier=evmhash,
+            sequence_index=1,
+            timestamp=TimestampMS(1665138455000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.STAKING,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.000033225735796413')),
+            location_label=user_address,
+            notes='Collect 0.000033225735796413 ETH from liquity\'s stability pool',
```

Comment:
For such strings is the only place where we prefer ""
```suggestion
            notes="Collect 0.000033225735796413 ETH from liquity's stability pool"',
```